package.json

```diff
@@ -141,17 +142,19 @@
     "prebundle": "npm run clean && npm run copyTestImages && npm run copyTopSitesJson",
     "bundle:static": "cpx \"content-src/static/**/*\" data/content",
     "bundle:webpack": "NODE_ENV=production webpack",
+    "bundle:webpackAddon": "webpack --config webpack.addon.config.js",
```

Comment:
in this case we don't since there are no production settings... might not be a bad idea for future proofing though